### PR TITLE
Update bug-report.md

### PR DESCRIPTION
**Pull request by bschnurr**
Mention ReportIssue command

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Bug Report
-about: 'Create a report to help us improve. '
+about: 'Create a report to help us improve. Auto fill reports with __Pylance: Report Issue__ command'
```
Comment (jakebailey) [inline .github/ISSUE_TEMPLATE/bug-report.md:3]: How about:

> Consider using the "Pylance: Report Issue" command to fill this out automatically.

Comment (savannahostrowski) [reply .github/ISSUE_TEMPLATE/bug-report.md:3]: "You can use the "Pylance: Report Issue" command to fill this out most of this information automatically."

Comment (bschnurr) [reply .github/ISSUE_TEMPLATE/bug-report.md:3]: i was trying to make it bold as well with the underscores. but not sure it was worked or not

Comment (bschnurr) [reply .github/ISSUE_TEMPLATE/bug-report.md:3]: I vote Jakes because its shorter

Comment (savannahostrowski) [reply .github/ISSUE_TEMPLATE/bug-report.md]: It doesn't actually fill out all the info though

Comment (bschnurr) [reply .github/ISSUE_TEMPLATE/bug-report.md]: ah good point

Comment (bschnurr) [reply .github/ISSUE_TEMPLATE/bug-report.md]: "Use the "Pylance: Report Issue" command to fill this out most of this information automatically."

how about slightly shorter?

Comment (bschnurr) [reply .github/ISSUE_TEMPLATE/bug-report.md]: or 
"The "Pylance: Report Issue" command can fill out most of this information automatically."

Comment (savannahostrowski) [reply .github/ISSUE_TEMPLATE/bug-report.md]: I prefer the former. 

"Use the "Pylance: Report Issue" command to fill this out most of this information automatically."

One nit is that this doesn't actually explain how to use this command. It'd be cool if we could make this more actionable and mention that you can grab it from the command palette.

Comment (bschnurr) [reply .github/ISSUE_TEMPLATE/bug-report.md]: "Use the "Pylance: Report Issue" command, from the command palette, to fill this out most of this information automatically."

Comment (savannahostrowski) [reply .github/ISSUE_TEMPLATE/bug-report.md]: "Use the "Pylance: Report Issue" command (via command palette) to fill this out most of this information automatically."

Comment (bschnurr) [reply .github/ISSUE_TEMPLATE/bug-report.md]: updated

**Review by savannahostrowski — APPROVED** [review]

**Pull request merged**